float64 posx
float64 posy
float64 posz
float64 posx_d
float64 posy_d
float64 posz_d
float64 euler_z
float64 euler_y
float64 euler_x
float64 euler_z_d
float64 euler_y_d
float64 euler_x_d
float64 ef_fx
float64 ef_fy
float64 ef_fz
float64 ef_tx
float64 ef_ty
float64 ef_tz
float64 velx
float64 vely
float64 velz
float64 wx
float64 wy
float64 wz
float64 accx
float64 accy
float64 accz
float64 betax
float64 betay
float64 betaz
float64 err_ori_x
float64 err_ori_y
float64 err_ori_z